packages/fxa-shared/subscriptions/types.ts

```diff
@@ -47,7 +47,7 @@ export interface ProductMetadata {
   emailIconURL?: string | null;
   playStoreLink?: string;
   productOrder?: string | null;
-  productSet?: string[] | null;
+  productSet: string[] | null;
```

Comment:
Could you remove the `null` here since it's no longer  valid?

Reply:
Interesting that there are several other fields that are `...?: string | null` which breaks down to `string | null | undefined`.